src/app/app.component.html

```diff
@@ -1,20 +1,12 @@
-<!--The content below is only a placeholder and can be replaced.-->
-<div style="text-align:center">
-  <h1>
-    Welcome to {{ title }}!
-  </h1>
-  <img width="300" alt="Angular Logo" src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCAyNTAgMjUwIj4KICAgIDxwYXRoIGZpbGw9IiNERDAwMzEiIGQ9Ik0xMjUgMzBMMzEuOSA2My4ybDE0LjIgMTIzLjFMMTI1IDIzMGw3OC45LTQzLjcgMTQuMi0xMjMuMXoiIC8+CiAgICA8cGF0aCBmaWxsPSIjQzMwMDJGIiBkPSJNMTI1IDMwdjIyLjItLjFWMjMwbDc4LjktNDMuNyAxNC4yLTEyMy4xTDEyNSAzMHoiIC8+CiAgICA8cGF0aCAgZmlsbD0iI0ZGRkZGRiIgZD0iTTEyNSA1Mi4xTDY2LjggMTgyLjZoMjEuN2wxMS43LTI5LjJoNDkuNGwxMS43IDI5LjJIMTgzTDEyNSA1Mi4xem0xNyA4My4zaC0zNGwxNy00MC45IDE3IDQwLjl6IiAvPgogIDwvc3ZnPg==">
-</div>
-<h2>Here are some links to help you start: </h2>
-<ul>
-  <li>
-    <h2><a target="_blank" rel="noopener" href="https://angular.io/tutorial">Tour of Heroes</a></h2>
-  </li>
-  <li>
-    <h2><a target="_blank" rel="noopener" href="https://github.com/angular/angular-cli/wiki">CLI Documentation</a></h2>
-  </li>
-  <li>
-    <h2><a target="_blank" rel="noopener" href="https://blog.angular.io/">Angular blog</a></h2>
-  </li>
-</ul>
+<h2>Trump's Speech</h2>
+<button type="button" (click)="displayText()">{{show ? 'Hide Details' : 'Display Details'}}</button>
 
+<p *ngIf="show">
+  We are going to make placeholder text great again. Greater than ever before.
+  Look at that text! Would anyone use that? Can you imagine that, the text of your next webpage?!
+  Look at that text! Would anyone use that? Can you imagine that, the text of your next webpage?!
+  I think the only difference between me and the other placeholder
+  text is that I’m more honest and my words are more beautiful.
+</p>
+
+<p *ngFor="let arr of arr; index as i;" [ngClass]="{'background-green' : i>=10}" [ngStyle]="{'color': i>=10 ? 'white' : 'black'}">{{ arr.time }} {{ arr.state }} {{ i }}</p>
```

Comment:
ideally, you should name the individual variable as something different than the array property name.